apps/wallet/src/ui/app/pages/home/nfts/index.tsx

```diff
@@ -127,14 +126,6 @@ function NftsPage() {
         }
     }, [ownedAssets]);
 
-    if (isLoading) {
```

Comment:
yep, I removed that because it was causing the layout to look weird, the loader is still showing, but now it’s appearing just as it should with the `<Loading>` component, like in the activity page :pray: :see_no_evil:

Reply:
Works great now, it shows the loader and the top heading
![image](https://github.com/user-attachments/assets/4d2dd899-9fb7-425a-b439-321b9b567aca)

Reply:
Works great now 
![image](https://github.com/user-attachments/assets/56d7c216-796e-4996-a52b-4f3ff8070a0a)